addons/mail_group/tests/test_mail_group.py

```diff
@@ -78,6 +78,51 @@ def test_find_member(self):
         member = self.test_group._find_member(email, partner_2.id)
         self.assertFalse(member, 'Should not return any member because the only one with the same email has a different partner')
 
+    def test_find_member_for_alias(self):
+        """Test the matching of a mail_group.members, when 2 users have the same partner email, and
+        that the first user was subscribed."""
+        User = self.env['res.users'].with_context(no_reset_password=True)
+        email = 'Alias <test_find_member_for_alias@example.com>'
+
+        alias = self.env['mail.alias'].create({
+            'alias_name': 'test_find_member_for_alias',
+            'alias_model_id': self.env.ref('base.model_res_partner').id,
```

Comment:
Why doing an alias on partner in a test for mail groups ?

Reply:
same that https://github.com/odoo/odoo/pull/80112#discussion_r756044110